projects/igniteui-angular/src/lib/card/card.component.ts

```diff
@@ -339,9 +358,7 @@ export class IgxCardActionsComponent implements OnInit, OnChanges {
     }
 }
 
-/**
- * @hidden
- */
+/** @hidden @internal */
```

Comment:
@mddragnev , according to the latest API docs guidelines, modules can't be `@internal`